recipes/sphinx-notfound-page/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "sphinx-notfound-page" %}
+{% set version = "0.5" %}
+{% set hash = "0ff34a26140ede859dc9bcc216107a5e27dcd0076a1b1defaa31f61fb67b489c" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - sphinx
+
+test:
+  imports:
+    - notfound
+    - notfound.extension
+
+about: 
+  home: https://github.com/readthedocs/sphinx-notfound-page 
+  license: MIT
+  license_file: LICENSE
+  summary: 'Create a custom 404 page with absolute URLs hardcoded'
+  doc_url: https://sphinx-notfound-page.readthedocs.io
+  dev_url: https://github.com/readthedocs/sphinx-notfound-page
+
+extra:
+  recipe-maintainers:
+    - astamminger
+    - chrisjsewell
```

Comment:
@chrisjsewell are you okay with being a maintainer on this feedstock

Reply:
Hi @mariusvniekerk

consent can be found here: https://github.com/conda-forge/aiida-core-feedstock/issues/12#issuecomment-779866063

Thanks! :)